Policies/github-terms-of-service.md

```diff
@@ -166,24 +197,28 @@ This Agreement is licensed under this [Creative Commons Zero license](https://cr
 ### H. API Terms
 **Short version:** *You agree to these Terms of Service, plus this Section H, when using any of GitHub's APIs (Application Provider Interface), including use of the API through a third party product that accesses GitHub.*
 
-Abuse or excessively frequent requests to GitHub via the API may result in the temporary or permanent suspension of your Account's access to the API. GitHub, in our sole discretion, will determine abuse or excessive usage of the API. We will make a reasonable attempt to warn you via email prior to suspension.
+Use of any of GitHub's APIs requires observance of the rest of this section. So long as you don't use the API, you may just ignore the rest of this section. 
```

Comment:
notices that I can ignore sections are just junk.

Reply:
Hello @KOLANICH.

Slightly confused here.

Can you please elaborate on 2?

In regard to not being allowed to ignore ToS sections, sometimes in contracts (if my memory serves me right), there can be text along the lines of 'this section only applies to API users'. This is the kind of thing I wanted to encode in the 'you can skip' texts. I'd like to add that something like this already exists in the ToS, where government users are bound by extra terms and non-government users can completely ignore those extra terms. I agree that the precise phrasing of these 'you can skip' texts perhaps needs to be re-thought. The important thing (in my mind) was marking the places where such 'you can skip' devices can be placed in the ToS. 

Regarding the first point, firstly, it appears to contradict your second point, so I'm a bit confused as to what you are trying to say. I've read very many contracts, including many contracts for websites and website services. My starting point for my proposed changes, was to try to make it easier for users to comply with the agreement. You can indeed ignore any piece of the ToS, but if those pieces include important user obligations, you may very badly fail in your contract compliance. What I've tried to do is highlight those parts that you can genuinely ignore without any detriment to your contract compliance.

Reply:
>You can indeed ignore any piece of the ToS, but if those pieces include important user obligations, you may very badly fail in your contract compliance.

Yes, but I can ignore them. I mean I can even not read them. You know, most of users when sees EULAs in sowtware installers just clicks a checkbox `I have read and aggree` and then `next` without reading. It is a lie, but it is possible to do. It is obvious that it can be done to website ToSes too. Since it is obvious there is no need to clutter the text with it: the goal of this PR is to make the ToS short in order to make it easier and **faster** to review and read.

>Can you please elaborate on 2?

If the ToS is accepted as whole it means that a one accepting it is required read, understand and aggree with all its parts.

P.S. I am not a lawer.

Reply:
Hello @KOLANICH,

Useful to get someone else's views on these things.

I've been thinking about such things for perhaps 5-10 years now, mostly because of my adopted religious beliefs.

Firstly, yes, you are right that many users just ignore user agreements. However, I'm of the opinion that the practices of businesses and consumers needs to be reformed such that there is a better respect of user agreements. My PR is part of that. I **feel** for other users reviewing GitHub's terms, liked how I had to review them.

One point about lying -> Have thought about it a great deal, and according to my definition of lying, it isn't necessarily lying. A lie is defined (by my religion), as the telling of a falsehood with the intention to deceive. Because users can click on 'I agree' without intending to deceive (any human being), I don't think it is necessarily a lie.

Your point about trying to avoid cluttering the text is valid. Perhaps ideally, the section headings would accurately define the content of the respective sections. That way, as an example, when users came to sections labelled 'API terms', they would know that they didn't need to read the section if they weren't going to use any of the APIs. In practice though, accurate section headings can be difficult to implement. If you look at these ToS, there is a specific disclaimer saying that section headings are not legally binding. However, I'll concede that if you think this idea of accurate section headings is doable, then perhaps we should go down that route. Otherwise, I stick to believing my 'you can skip' devices, or at least some equivalent variation of them, is the right way forward.

Regarding what acceptance of a ToS means, I don't quite agree with you. I don't believe agreement to a ToS means that you have read them. I believe you can agree to the terms of an agreement without having read them. Agreement can mean that you agree to be open to penalty in accordance with the terms of an agreement. It's true that sometimes you are required to make a statement that you have read a certain ToS. I don't like such things, and I generally think they're a bit stupid. For example, in the days of AI, we will quite likely soon be able (or already are able) to get AI bots to read these contracts, at least in part. We can also enlist the help of other humans in the reading. Finally, as hinted above, I don't think you necessarily need to read agreements at all (depending upon the circumstances).

The subject of user agreements is quite a large topic. My PR is just one step in hopefully improving the current state of user agreements in general.